easybuild/easyconfigs/p/PEcAn/PEcAn-1.7.2.9000-foss-2023a-R-4.3.2.eb

```diff
@@ -0,0 +1,218 @@
+    }),
+    ('PEcAn.settings', '1.7.2', {
+        'checksums': ['ab0dd41e9f421a84e68a16a1e4b290f70d852d23e66a2de1c3e91bf883aa1aeb'],
+    }),
+    ('PEcAn.MA', '1.7.2', {
+        'checksums': ['bf3a5b6ac8adce4efb6be504cdeb10a71a59b3ac5af72c8d77b59a39c7d90ceb'],
+    }),
+    ('PEcAn.emulator', version, {
+        'checksums': ['f4465a154e41cfe5a2cc94035ae4a5ddbfd3b5fc54a83bc9578c98a9eb07cff1'],
+    }),
+    ('PEcAn.priors', '1.7.2', {
+        'checksums': ['001c26275ef23dda369fa2a5b408e481265d29137edeaebd03a30a39b4682edd'],
+    }),
+    ('PEcAn.uncertainty', version, {
+        'checksums': ['a0d1a3a3d65e06b066d23473def763f5e158b2356ee5fe9e09020fe311e9ef4a'],
+    }),
+    ('amerifluxr', '1.0.0', {
+        'checksums': ['ec54dce4b8e4108832eaadb60f7ef8ad28b459d3381316537492dcd755d83b25'],
+    }),
+    ('geonames', '0.999', {
+        'checksums': ['1dd7bbd82d9425d14eb36f8e5bf431feaccfe3b0c4e70bf38f44f13dfc59e17b'],
+    }),
+    ('solartime', '0.0.2', {
+        'checksums': ['5dd7a28db2bd4881ab740cd7004b88cd05c61e0954e3a10c8e18ebbe07d3e7d1'],
+    }),
+    ('bigleaf', '0.8.2', {
+        'checksums': ['e41d82fb6298150c46f57128e959a1f1c6cfa55e916c32544bab7be69b0b6df7'],
+    }),
+    ('REddyProc', '1.3.3', {
+        'checksums': ['3f673bd24f41813f631bf25139c9595bd91483e898ec31b7207c4eb8b7ff486b'],
+    }),
+    ('suntools', '1.0.0', {
+        'checksums': ['42b7b39bd23ad48ded002150a0e7798fab94ed9828e5b5fdd8e5e1c9040697e2'],
+    }),
+    ('fauxpas', '0.5.2', {
+        'checksums': ['84d07361a146b419cb1dcb7751654e3d5f159cf2704faf498e5ea56c8de61ef6'],
+    }),
+    ('nneo', '0.1.0', {
+        'checksums': ['ca503e78cf4f4c85de6e18ad3ffde0c08d6a507ff5dc5ff91a50b3e04bf84a49'],
+    }),
+    ('PEcAn.data.atmosphere', '1.7.2', {
+        'checksums': ['1847151588f669e91b35de637c804328efbc606141bf992bee650fd261b70e5a'],
+    }),
+    ('redland', '1.0.17-18', {
+        'checksums': ['e999ba22321733df7ed056f528e38fc4be535d36bbac8106395c14f9f60a75a0'],
+    }),
+    ('datapack', '1.4.1', {
+        'checksums': ['6149733ae90e6bbbb53d5f85ca29cec5f811825e794b56029b7f325de6637493'],
+    }),
+    ('dplR', '1.7.7', {
+        'checksums': ['0e35a4d2a22773499f9e4592d1a198a08c9842388ce1e5a35394990ed4cdb2bd'],
+    }),
+    ('neonUtilities', '2.4.2', {
+        'checksums': ['37b4efd0ed3b724de999d08d4d430954a6c717b1d943bc84c513c4fe8dbca057'],
+    }),
+    ('storr', '1.2.5', {
+        'checksums': ['4224c3991d9c043a45ce530d0698d7f2cdca231b26fe31b45e0db865026e5f63'],
+    }),
+    ('thor', '1.1.5', {
+        'checksums': ['99802d8c8471ce351403f8428c1efab48ccf20f359fbd368db84f25ef8f7ecae'],
+    }),
+    ('duckdb', '1.0.0-2', {
+        'checksums': ['3735d51f22d62a7411aa514142cd569ce28bdaec4eb58d5f0893a476b9ccbf01'],
+        # fixes the `prefix_back_ was not declared` error, see https://github.com/duckdb/duckdb/pull/12344
+        'preinstallopts': "sed -i '%s' src/duckdb/third_party/re2/re2/prog.cc && " % local_sed_patterns_duckdb,
```

Comment:
@PetrKralCZ Why not use a patch file here? (see https://patch-diff.githubusercontent.com/raw/duckdb/duckdb/pull/12344.diff)

Reply:
fixed in bed9e08